src/odemis/gui/cont/tabs.py

```diff
@@ -1796,6 +1796,15 @@ def __init__(self, name, button, panel, main_frame, main_data):
             calibrations
         )
 
+        # Controller for acquistion settings panel
+        self._acq_settings_controller = settings.SettingsController(
+            panel.pnl_acq_settings,
+            "dwell time"
+        )
+        dt_conf = {"range": [1e-6, 40e-6]}  # Limit the values the user can set the dwell time to.
```

Comment:
Instead of putting this value here, I'd suggest to put it with the rest, in gui/conf/data.py/HW_SETTINGS_CONFIG["multibeam"]["dwellTime"]. 
Then, you'd access it with `get_hw_config(tab_data.main.multibeam).get("dwellTime")`. 
It's a little bit more verbose, but allows to keep that setting config in the same place as the others.